libs/shared/design-tokens/generated/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Please add a comment that this file is autogenerated and should not be modified :)